index.html

```diff
@@ -382,7 +382,12 @@ <h4 class=alert-heading>Lidarr v0.8 migration</h4>
                                 Download App package
                             </dt>
                             <dd>
-                                <a class="btn btn-lidarr" href=https://lidarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=x64&installer=true>Download macOS App</a>
+                                <div>
+                                    <a class="btn btn-lidarr" href=https://lidarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=x64&installer=true>Download Intel macOS App</a>
+                                </div>
+                                <div>
+                                    <a class="btn btn-radarr" href=https://lidarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=arm64&installer=true>Download Arm macOS App</a>
```

Comment:
I think the class here should be `btn-lidarr`

Reply:
Doh! Copy paste error!